### ISSUE TITLE
Feedback requested on new Pangeo website

### ISSUE THREAD

**Issue by rabernat** [pangeo-data/pangeo#229]
I have merged #28 and there is now a new pangeo website at https://pangeo-data.github.io/pangeo/

Please use this issue to give [reasonable, actionable] feedback on the new design and content. Thanks to @meganfitzsimons for creating the Pangeo branding.

The site is incomplete. Here is a todo list of items to complete before we can launch it.

- [x] **update the [setup guides](https://pangeo-data.github.io/pangeo/setup_guides/index.html) with the latest state of the art**
- [x] **produce an accurate and complete list of [active deployments](https://pangeo-data.github.io/pangeo/deployments.html)** 
- [x] write the page on [Technical Architecture](https://pangeo-data.github.io/pangeo/architecture.html) (contributions welcome)
- [x] fill in the section on [Pangeo affiliated packages](https://pangeo-data.github.io/pangeo/packages.html#pangeo-affiliated-packages)
- [x] complete the institutions list in the [collaborators page](https://pangeo-data.github.io/pangeo/collaborators.html#institutions)
- [x] finish the [guide for scientists](https://pangeo-data.github.io/pangeo/quickstart.html#contribute-a-use-case)
- [x] add more content to the [FAQ](https://pangeo-data.github.io/pangeo/faq.html)
- [x] figure out how to organize the [use case gallery](https://pangeo-data.github.io/pangeo/use_cases/index.html) (see #201)
- [x] figure out what to say about the data catalog (see #39)
- [ ] write a page on related projects

What else needs to be on this list?

This is all a lot of work. While your suggestions and advice are much appreciated, even more useful are PRs which implement those suggestions. The documentation is built in sphinx and lives in this repo under the `docs/` folder.

Especially seeking feedback from @jhamman, @kmpaul, @mrocklin, @jacobtomlinson, @niallrobinson, @naomi-henderson, @chiaral.

**Comment by jhamman** [comment]
@rabernat - I'm happy to help with the setup guides and to provide the NCAR deployments. Can you summarize the plan for the blog? Will it move to sphinx or Medium?

**Comment by rabernat** [comment]
I would like to move the blog to Medium as I think this is the best way to get our message out. I basically want to copy Jupyter and https://blog.jupyter.org/.

To make this work, it would be great to have a pangeo domain name. But most of the obvious ones are taken.

**Comment by rabernat** [comment]
pangeo.pizza?

https://domains.google.com/registrar?s=pangeo&hl=en&_ga=2.181994660.200793118.1525187629-1175578531.1522588763&_gac=1.253890236.1525187629.EAIaIQobChMItqrGy-bk2gIVw2SGCh0X0wVCEAAYASAAEgK4WPD_BwE#st=a&chp=st

**Comment by jhamman** [comment]
what about pangeo-data.com or pangeo-data.org or pangeo.golf 😉

**Comment by jacobtomlinson** [comment]
Good stuff! My initial aesthetic thoughts are that it looks really nice.

I'll have a proper look over this when I have more time but I expect there are lots of areas here I can contribute.

**Comment by pbranson** [comment]
pangeo.io is another option

**Comment by pbranson** [comment]
Actually ignore that, it is gone!

**Comment by mrocklin** [comment]
First view of the first page is just the logo and otherwise low on content.  The user needs to scroll below the fold to see something (or at least on my monitor resolution)

**Comment by tjcrone** [comment]
I like it. Also agree that the very light green links are washed out. A touch darker might be better. And would you want to have a link highlight color that fits with the scheme? Or, hot pink is also great.

pangeo.org is not being served. could be a squatter. see how much they want for it?

**Comment by tjcrone** [comment]
Nice sleuthing. I saw the WHOIS and just figured the name was a stand-in at godaddy. Very interesting.

**Comment by rabernat** [comment]
@kmpaul, can I convince you to reach out to Joe Sexton to ask if he will release the domain to us?

**Comment by kmpaul** [comment]
Sure!  I'll give it a shot.

**Comment by rabernat** [comment]
@kmpaul: any luck with the domain name?

**Comment by kmpaul** [comment]
Not yet, unfortunately.  I've contacted Joe Sexton, and he sounded (in his response) pretty interested in the Pangeo project.  However, he was traveling when I contacted him, and he said that he would get back to me sometime later this week.  If I have not heard back, I will reconnect with him and try to get an answer on the domain name.

**Comment by tjcrone** [comment]
Any updates on this @kmpaul? Would be great to figure out a domain name. I have been using a different domain name for my Azure-based system, and I am finding the use of subdomains to host separate Pangeo deployments using different images for different projects to be very useful. Something main Pangeo might be interested in exploring.

**Comment by kmpaul** [comment]
I'm still in very slow discussions with Joe Sexton.  After Joe got back from traveling, I touched base with him again, and he asked, "What's your offer?"  Unfortunately, he replied while I was on vacation, so I didn't get back to him until later last week.  I told him I'd be willing to buy him out of the rest of his "contract."  I haven't heard back.

**Comment by rabernat** [comment]
Architecture is done in #280.

**Comment by rabernat** [comment]
scientist guide done in #289.

use case updates in #290, #291, #292

**Comment by rabernat** [comment]
Punted on affiliated packages in #295

**Comment by rabernat** [comment]
New site is live at http://pangeo-data.org!
